data_diff/dbt_parser.py

```diff
@@ -209,6 +211,25 @@ def get_dbt_selection_models(self, dbt_selection: str) -> List[str]:
         logger.debug(str(results))
         raise DataDiffDbtSelectUnexpectedError("Encountered an unexpected error while finding `--select` models")
 
+    def get_simple_model_selection(self, dbt_selection: str):
+        model_nodes = dict(filter(lambda item: item[0].startswith("model."), self.dev_manifest_obj.nodes.items()))
+
+        model_unique_key_list = [k for k, v in model_nodes.items() if v.name == dbt_selection]
+
+        # name *should* always be unique, but just in case:
```

Comment:
by default, dbt prevents duplicate model IDs
![image](https://github.com/datafold/data-diff/assets/19176976/5d98ff16-4b8f-45f0-82b0-ec51b611e88e)

Reply:
even if they write to different schemas?

Reply:
It should still prevent it
![image](https://github.com/datafold/data-diff/assets/19176976/e91ad0f5-422e-4276-a3b8-3a59b27124d7)

Reply:
To be clear, no changes suggested here. Just calling this out to remove doubts on error handling.

Reply:
oh yep I see
> dbt found two models with the name

That should cover it